pymtl3/passes/backends/generic/test/RTLIRTranslator_L1_cases_test.py

```diff
@@ -5,59 +5,52 @@
 # Date   : May 23, 2019
 """Test the RTLIR transaltor."""
 
-from pymtl3.datatypes import Bits1, Bits32
-from pymtl3.dsl import Component, InPort, OutPort, connect
-from pymtl3.passes.rtlir.util.test_utility import do_test, expected_failure
+import pytest
 
-from ..behavioral.test.BehavioralTranslatorL1_test import *
+from pymtl3.passes.rtlir.util.test_utility import expected_failure, get_parameter
+
+from ..behavioral.test.BehavioralTranslatorL1_test import (
+    test_generic_behavioral_L1 as behavioral,
```

Comment:
Do you really want this "as" here? As far as I've seen they are only used once, so I think just import that test and use it below seems fine.

Reply:
Right they are used only once as a short hand..